dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -139,16 +138,90 @@ export const Card = ({
 	showAge = false,
 	discussionId,
 }: Props) => {
-	const palette = decidePalette(format, containerPalette);
-
 	const hasSublinks = supportingContent && supportingContent.length > 0;
-	const noOfSublinks = (supportingContent && supportingContent.length) || 0;
-
+	const noOfSublinks = supportingContent?.length ?? 0;
 	const isOpinion =
 		format.design === ArticleDesign.Comment ||
 		format.design === ArticleDesign.Editorial ||
 		format.design === ArticleDesign.Letter;
 
+	const cardIsHorizontal =
+		imagePosition === 'left' || imagePosition === 'right';
```

Comment:
In theory we could have `imagePosition` set to `top`, while `imagePositionOnMobile` was set to `left` or `right`. If I've understood correctly then this card would not count as `cardIsHorizontal`. Is that the behaviour we want (i.e. only triggering `isHorizontal` above mobile)? Or does the decision logic need to take this case into account?

(actually, it looks like we have at least one actual case, although it's in code that I wrote last week so it's possible that I violated an assumption without realising? https://github.com/guardian/dotcom-rendering/blob/d54ec25b49c5be77aec0743a7d76c6ad3bd8f507/dotcom-rendering/src/web/components/FixedSmallSlowIII.tsx#L51-L52 )